(MODULES-2180) Fix the max value of RebootRelaunchTimeout

digglife opened the pull request:
The max value of RebootRelaunchTimeout should be 1440(24 hours), rather than 440. I guess this miss is due to the numerical separator used in the [official document](https://technet.microsoft.com/en-us/library/dd939844%28v=ws.10%29.aspx). 

```
Range = n, where n = time in minutes (1–1,440). 
```

digglife commented:
Modified the test file a bit  to reflect the changes. It passed the [build test](https://ci.appveyor.com/project/digglife/puppetlabs-wsus-client/build/1.1.x.2).

ferventcoder commented:
Hello @digglife - this is great! Would you mind filing a ticket for this at https://tickets.puppetlabs.com/browse/MODULES? Then followup with the commits with that ticket number (and update this PR title as well)?  Please take a look at https://github.com/puppetlabs/puppetlabs-wsus_client/blob/master/CONTRIBUTING.md for more information on how we expect the general flow of things to go. 

Thanks!

digglife commented:
@ferventcoder Actually I tried to raise a ticket at first, but there is no such `wsus_client` component in the issue form, so I gave up and ended up creating pull request here. It seems that I could have left the component blank... :blush: I rebased them to one single commit and changed the message a bit, hope it's OK.

cyberious commented:
:+1:

cyberious commented:
Thanks @digglife released 0.1.3
